src/dotnet/install.sh

```diff
@@ -111,6 +117,10 @@ for version in "${aspNetCoreRuntimeVersions[@]}"; do
     install_runtime "aspnetcore" "$version"
 done
 
+for workload in "${workloads[@]}"; do
```

Comment:
Can we optimize the changes in this file? (untested)

```
IFS=',' read -ra workloads <<< "$WORKLOADS"
for workload in "${workloads[@]}"; do
    if [[ -n "$workload" ]]; then
        install_workload "$workload"
    fi
done
```

Reply:
I changed the code to run `dotnet workload install <workload_id1> [<workload_id2> ...]` in a single invocation

Reply:
Superb!